src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    public static Connection getConnection() {
+        try {
+            String className = "com.mysql.cj.jdbc.Driver";
+            Class.forName(className);
+            Properties dbProperties = new Properties();
+            dbProperties.put("user", "root");
+            dbProperties.put("password", "HQaHPS");
```

Comment:
Hardcoding database credentials in the source code is not secure. Consider using environment variables or a configuration file to store the database username and password.